front_end/src/components/history/history.css

```diff
@@ -235,3 +253,12 @@
   top: 45px;
   text-align: center;
 }
+
+.rewardsPointsBalance {
+  color: #3d4e96;
+  text-align: right;
+  font-size: 20px;
+  font-weight: 600;
+  margin-bottom: 0px;
+  margin-right: 25px;
+}
```

Comment:
Couldn't highlight it but there is a class named .HistoryPageDates and another called .HistoryContainerExpand both aren't being used.